src/siteSettings/components/SiteSettingsPage/SiteSettingsPage.tsx

```diff
@@ -15,8 +15,8 @@ import useStateFromProps from "@dashboard/hooks/useStateFromProps";
 import { commonMessages } from "@dashboard/intl";
 import createSingleAutocompleteSelectHandler from "@dashboard/utils/handlers/singleAutocompleteSelectChangeHandler";
 import { mapCountriesToChoices } from "@dashboard/utils/maps";
-import { ConfirmButtonTransitionState, makeStyles } from "@saleor/macaw-ui";
-import { Box } from "@saleor/macaw-ui/next";
+import { ConfirmButtonTransitionState } from "@saleor/macaw-ui";
```

Comment:
**suggestion**: I saw in @poulch PR that we migrated `ConfirmButton` to new MacawUI in dashboard. 

@poulch has this PR been merged?